### DISCUSSION TITLE
can mitosis compile to plain html?

### DISCUSSION THREAD

**Discussion by pmbanugo** [BuilderIO/jsx-lite#1582] [General]
can mitosis compile to plain html, possible use in a multi-page app?

**Comment by samijaber** [comment]
Mitosis does have an HTML generator. it's not showing in the new playground but you can see it in the old one


![CleanShot 2024-10-04 at 09 52 14@2x](https://github.com/user-attachments/assets/6339a818-486d-4dcf-a13d-e524687a6d51)
